extensions/typescript-language-features/src/languageFeatures/quickFix.ts

```diff
@@ -366,30 +366,27 @@ class TypeScriptQuickFixProvider implements vscode.CodeActionProvider<VsCodeCode
 			let expand: Expand | undefined;
 			let title = action.description;
 			if (action.fixName === fixNames.classIncorrectlyImplementsInterface) {
-				title += ' with Copilot';
+				title = vscode.l10n.t('{0} with AI', action.description);
 				message = `Implement the stubbed-out class members for ${document.getText(diagnostic.range)} with a useful implementation.`;
```

Comment:
@mjbvz should this `message` on line 370 be wrapped with `vscode.l10n.t`?

(I'm not familiar with the localization, I was just being nosey and noticed that all the others were updated but not this one)

Reply:
Yes thanks should be localized as well